### PR TITLE
[FIX] patch tables for setup_models()

### PR DESCRIPTION
**Pull request by coleste**
Odoo does not boot unless some tables are manually updated on the migrated database. As discussed in #404 .

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Seems OK and can be run multiple times without breaking, so :+1:

**Comment by hbrunn** [comment]
isn't this too late? Why not in `registry.__init__`? This way, it's run at https://github.com/OCA/OpenUpgrade/blob/8.0/openerp/modules/loading.py#L133

**Comment by coleste** [comment]
It crashes on cr.execute('select model, transient from ir_model where state=%s', ('manual',)), 3 lines below. But yes, it could be moved right after the has_unaccent bit. Should I? Done.

@hbrunn is this better?

**Comment by mmalorni** [comment]
:+1:

File: openerp/modules/registry.py [hunk 1]
```diff
@@ -65,6 +65,9 @@ def __init__(self, db_name):
         if openerp.tools.config['unaccent'] and not has_unaccent:
             _logger.warning("The option --unaccent was given but no unaccent() function was found in database.")
         self.has_unaccent = openerp.tools.config['unaccent'] and has_unaccent
+
+        # ir_model must be in 9.0 format before continuing
```
Comment (StefanRijnhart) [inline openerp/modules/registry.py]: Can you add a marker #OpenUpgrade to this change?

File: openerp/modules/registry.py [hunk 2]
```diff
@@ -172,6 +175,34 @@ def load(self, cr, module):
 
         return [self.models[m] for m in models_to_load]
 
+    def patch_migrate_9(self, cr):
```
Comment (StefanRijnhart) [inline openerp/modules/registry.py]: Can you put this in a new module odoo/openerp/openupgrade/openupgrade_loading_90.py?

**Comment by StefanRijnhart** [comment]
Thanks, looks good! Just a request to isolate the code into the OpenUpgrade corner as much as possible.

**Comment by StefanRijnhart** [comment]
Thanks for the update. Next time, please write a comment that you followed up the comments, so that the reviewers get notified by email.
:+1:

**Comment by StefanRijnhart** [comment]
Fixed trivial flake8 issue in direct commit https://github.com/OCA/OpenUpgrade/commit/cbced5496c81f9d328da408f27751471e78aa73f

**Comment by coleste** [comment]
I will, and thanks for the whitespace!

**Comment by coleste** [comment]
@StefanRijnhart , I missed something and had to resubmit as #410 
I missed the commit()

**Pull request merged**